fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1676,6 +1689,8 @@ void ReadYourWritesTransaction::clear( const KeyRef& key ) {
 	}
 	
 	KeyRangeRef r = singleKeyRange( key, arena );
+	approximateSize += r.expectedSize() + sizeof(KeyRangeRef) +
+	                   (addWriteConflict ? sizeof(KeyRangeRef) + 2 * key.expectedSize() + 1 : 0);
```

Comment:
Since you've already created your range, you could alternately do something like this:

```suggestion
	                   (addWriteConflict ? sizeof(KeyRangeRef) + r.expectedSize() : 0);
```